app/src/interfaces/select-radio/select-radio.vue

```diff
@@ -38,19 +39,25 @@ const gridClass = computed(() => {
 	if (choices?.value === undefined) return null;
 
 	const widestOptionLength = choices.value.reduce((acc, val) => {
-		if (val.text.length > acc.length) acc = val.text;
+		if (stringWidth(val.text) > acc) acc = stringWidth(val.text);
```

Comment:
I think we do a similar piece of logic in the checkboxes interface as well. Mind checking that? Would be nice if we can squash to birds with one stone 🙂

Reply:
Also can we store it in a variable so we don't have to run it twice?

Reply:
Thanks for suggestions!
I have add a util function and test case.